src/catalog/catalog.scss

```
@@ -0,0 +1,54 @@
+.calendar {
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @include square-size($box-length);
+    background-color: #eee;
+    border: 1px solid #000;
+  }
+
+  &__day:hover {
+    background-color: #ffc0cb;
+    cursor: pointer;
+    transition-duration: 500ms;
```

```suggestion
    transition-duration: 500ms;
```

move to day to add effect on onFocus